### PR TITLE
Feature/start end time reversed

### PR DESCRIPTION
**Pull request by wshands**
Fix the reversed download start and stop times and the reversed compute start and stop times and organize the times as: start, stop, total time

### CODE REVIEW

**Comment by wshands** [comment]
Carlos, I need to test this first

**Comment by wshands** [comment]
Tested on my dev box and looks OK

File: DockstoreRunner.py [hunk 1]
```diff
@@ -513,18 +513,18 @@ def run(self):
    "timing_metrics" : {
       "step_timing" : {
          "download" : {
+            "start_time_utc" : "%s"
```
Comment (caaespin) [inline DockstoreRunner.py]: Commas here, otherwise the indexer will break

File: DockstoreRunner.py [hunk 2]
```diff
@@ -513,18 +513,18 @@ def run(self):
...
             "stop_time_utc" : "%s",
             "walltime_seconds" : %d,
-            "start_time_utc" : "%s"
          },
          "tool_run" : {
+            "start_time_utc" : "%s"
```
Comment (caaespin) [inline DockstoreRunner.py]: Commas, same as above

**Review by caaespin — APPROVED** [review]
LGTM

**Pull request merged**